client/src/includes/breadcrumbs.js

```diff
@@ -15,6 +15,10 @@ export default function initCollapsibleBreadcrumbs() {
     '[data-toggle-breadcrumbs]',
   );
 
+  if (!breadcrumbsToggle) {
+    return;
+  }
+
```

Comment:
this should no longer be needed - we can add this logic if the breadcrumbs get used in other areas of the code later.